src/js/plugin/external/PluginActions.js

```diff
@@ -0,0 +1,7 @@
+import Util from "../../helpers/Util";
+
+const PluginActions = Util.objectCreateWithAdder("PLUGIN_ACTION_");
+
+["DIALOG_ALERT"].forEach(PluginActions.add);
```

Comment:
Do you think, there is way to define the default const as "normal" object properties? I'm thinking of something like the following: 

```
const PluginActions = {
	INTIAL_FIXED_ACTION_PROPERTY: Symbol()
}

// Fix properties (kinda like a frozen/sealed object but still extensible
export default Util.fix(PluginActions);

```

```
PluginActions.WRITABLE_ACTION_PROPERTY = "insecure"
PluginActions.addFixedProperty("FIXED_ACTION_PROPERTY_WITH_DEFAULT_VALUE");
PluginActions.addFixedProperty("FIXED_ACTION_PROPERTY_WITH_CUSTOM_VALUE", "custom");
```